build.savant

```diff
@@ -47,18 +48,18 @@ project(group: "org.savantbuild", name: "savant-dependency-management", version:
 }
 
 // Plugins
-dependency = loadPlugin(id: "org.savantbuild.plugin:dependency:2.0.0-RC.6")
-java = loadPlugin(id: "org.savantbuild.plugin:java:2.0.0-RC.6")
-javaTestNG = loadPlugin(id: "org.savantbuild.plugin:java-testng:2.0.0-RC.6")
-idea = loadPlugin(id: "org.savantbuild.plugin:idea:2.0.0-RC.6")
-release = loadPlugin(id: "org.savantbuild.plugin:release-git:2.0.0-RC.6")
+def dependency = loadPlugin(id: "org.savantbuild.plugin:dependency:2.0.0-RC.7")
```

Comment:
These aren't really needed for a DSL and we don't use them anywhere else. Not sure I want to start adding `def` to every build file. Thoughts?

Reply:
Reverted after talking with Daniel. I did this mainly for IntelliJ syntax highlighting but there are other ways to accomplish that.